p2p/tests/identify.rs

```diff
@@ -268,7 +268,7 @@ fn bad_node_effects(
                 event_mapper_effect(store, a);
             }
             Action::Idle(_) => {
-                p2p_timeout_effects(store, &meta);
+                // p2p_timeout_effects(store, &meta);
```

Comment:
Leftover comment or required?

Reply:
I need to update this and kademlia bad node tests after I update p2p testing framework